spec/models/html_document_spec.rb

```diff
@@ -213,6 +213,34 @@
 
       it { is_expected.to eq 'http://www.foo.gov/og_image.jpg' }
     end
+
+    context 'when og:image is not available' do
+      let(:raw_document) do
+        <<~HTML
+          <html lang="en">
+            <head>
+              <meta name="og:image" content="">
```

Comment:
@krbhavith , I was actually hoping for a spec where the `og:image` meta tag isn't present at all. That possibility is the reason for the safe navigation operator, because without the `og:image` tag, this would fail:
```ruby
url.match?(URI::DEFAULT_PARSER.make_regexp)
```
while this would not:
```ruby
url&.match?(URI::DEFAULT_PARSER.make_regexp)
```